### PR TITLE
Opifex

### PR DESCRIPTION
**Pull request by Lamasmaster**
## About The Pull Request
<details>
<summary>
	About The Pull Request
</summary>
<hr>

Nerfs starting equipment from perks, gives them more mechanical on roundstart, now they can in agressive grab tear apart enemy (on cooldown) and leap onto enemies.
	
<hr>
</details>

## Changelog
:cl:
tweak: Opifex
balance: Opifex
/:cl:

### CODE REVIEW

**Comment by Trilbyspaceclone** [comment]
what does this pr do?
Like for changelogs idk what to put down for it

File: code/game/objects/items/weapons/storage/belt.dm [hunk 1]
```diff
@@ -376,18 +372,15 @@
 	desc = "A black tactical belt made specifically for opifex to prevent any pulling or ruffling of feathers, slightly uncomfortable for anyone else but none can deny its quality. This harness is built specifically for combat, limiting its versatility."
 
 /obj/item/storage/belt/security/tactical/opifex/full/populate_contents()
-	new /obj/item/reagent_containers/spray/pepper(src)
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/weapons/storage/belt.dm:379]: why no pepper spray?

File: code/game/objects/items/weapons/storage/belt.dm [hunk 2]
```diff
@@ -194,17 +193,14 @@
...
+	new /obj/item/reagent_containers/syringe/tricordrazine(src)
+	new /obj/item/reagent_containers/syringe/inaprovaline(src)
 	new /obj/item/storage/pill_bottle/tramadol(src)
-	new /obj/item/storage/pill_bottle/prosurgeon(src)
 	new /obj/item/storage/pill_bottle/bicaridine(src)
 	new /obj/item/storage/pill_bottle/dermaline(src)
 	new /obj/item/storage/pill_bottle/dexalin_plus(src)
 	new /obj/item/storage/pill_bottle/antitox(src)
 	new /obj/item/bodybag/cryobag(src)
-	new /obj/item/extinguisher/mini(src)
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/weapons/storage/belt.dm:207]: I dont understand why this is being removed

Comment (Lamasmaster) [reply code/game/objects/items/weapons/storage/belt.dm:207]: You can get this in a lathe

File: code/game/objects/items/weapons/storage/belt.dm (hunk 1 above)
Comment (Lamasmaster) [reply code/game/objects/items/weapons/storage/belt.dm:379]: You can just buy this for like 50 credits

Comment (Trilbyspaceclone) [reply code/game/objects/items/weapons/storage/belt.dm:379]: Id rather have this be kept

File: code/game/objects/items/weapons/storage/belt.dm (hunk 2 above)
Comment (Trilbyspaceclone) [reply code/game/objects/items/weapons/storage/belt.dm:207]: You can get pill bottles, syringes and other things in lathes, thats not a reason to remove an item from this list

**Pull request merged**